velox/docs/develop/debugging/print-plan-with-stats.rst

```diff
@@ -208,14 +249,14 @@ TableScan also reports the number of splits assigned.
 
 .. code-block::
 
-	Raw Input: 20480 rows (72.31KB), Splits: 20
+	Raw Input: 20480 rows (72.79KB), Splits: 20
 
-Velox also measures CPU time and peak memory usage for each operator. This
-information is shown for all plan nodes.
+Velox also measures CPU time and the breakdown of CPU time which including addInput, getOutput and finish time,
```

Comment:
There are some typos. Overall, it might be better to split this paragraph.

```
Velox also measures CPU time, peak memory usage and total number of memory allocations for each operator. 

Cpu time: 8.89ms, Peak memory: 80.38KB, Memory allocations: 262

A breakdown of CPU time into addInput, getOutput and finish stages of the operator is also available. I/O/F below is a shortcut for addInput/getOutput/finish.

CPU breakdown: I/O/F (0ns/8.88ms/4.93us)
```